### DISCUSSION TITLE
Calculate area given a coplanar pointcloud

### DISCUSSION THREAD

**Discussion by pm222** [isl-org/Open3D#6173] [Q&A]
Hi! 
I have a pointcloud which contains coplanar 3d points. I'd like to calculate the surface area what could cover these points. How should I start? I tried to add mesh to the pointcloud with TriangleMesh.create_from_point_cloud_ball_pivoting, but the result is not satisfying, and the mesh area is wrong because of the errors of the mesh. Examples uploaded. 
Pointcloud:
![image](https://github.com/isl-org/Open3D/assets/97162296/7674e31e-b52c-45ad-add9-cac71a2f4abd)
Mesh: 
![image](https://github.com/isl-org/Open3D/assets/97162296/7c8e3bab-dac8-41ef-a245-34a60e290c7d)

**Comment by benjaminum** [comment]
Are these points aligned to a grid?
